hibernate-core/src/main/java/org/hibernate/persister/collection/BasicCollectionPersister.java

```diff
@@ -502,7 +526,8 @@ private void applyUpdateRowValues(
 				jdbcValueBindings,
 				null,
 				(valueIndex, bindings, y, jdbcValue, jdbcValueMapping) -> {
```

Comment:
I think it's easier to understand this way.

```suggestion
				hasIndex()
				? (valueIndex, bindings, y, jdbcValue, jdbcValueMapping) -> {
					if ( jdbcValueMapping.isFormula() ) {
						return;
					}
					bindings.bindValue(
							jdbcValue,
							jdbcValueMapping,
							ParameterUsage.SET
					);
				}
				: (valueIndex, bindings, y, jdbcValue, jdbcValueMapping) -> {
					if ( !jdbcValueMapping.isUpdateable() ) {
						return;
					}
					bindings.bindValue(
							jdbcValue,
							jdbcValueMapping,
							ParameterUsage.SET
					);
				},
				session
		);
```

Reply:
done :)